src/index.html

```diff
@@ -27,15 +27,15 @@
       </section>
 
       <section class="item">
-        <div class="item__name">
+        <h1 class="item__name">
```

Comment:
h1 can be only one at page, in next task you will create more than one card, change it